### ISSUE TITLE
Release 54 does not work for me

### ISSUE THREAD

**Issue by tringate** [johanneszab/TumblThree#88]
I updated from release 48 to 54 in my usual way.  I deleted all files for the program except for the .json files which there are three of.

Then loaded the new release into the folder, launched the program and verified I was at release 54.

Everything worked fine in selecting a specific blog, adding it to the queue.  Then when I hit crawl, it changes to the blue color in the queue listing and the activity bar is green indicating it is working with that blog.

Nothing happens, it never tries to download anything.  I added a second blog to the queue which also appeared in the queue list and turned blue indicating it was ready for download.  Nothing happened with either blogs.

Then my tumblrthree window locked and the only way to remove it was to go to task manager and kill the task.

I did a system reboot, tried several times to get release 54 to do something with no success.  Dropped back to release 48 and everything worked just fine.

Not sure what else I can tell you about the issue.

Am I doing the update procedure correctly?  It has always worked for me in the past.

I do have a question about the note on the release 48 download.  You use what looks like a hot key to select all blogs, but I do not know what "Simply select all blogs (strg-a)" means.  Is strg-a a hot key sequence like ctrl-a?

BTW, release 54 has the file date correction that is going to make life so much simpler for me.  Thanks for adding that change.

**Comment by johanneszab** [comment]
You probably forgot to set the _posts per page_ size. As I've understood it you've kept your settings from the previous release. I always update the defaults settings internally but I don't force any update on them if there already is a settings file.
I'm not happy with the user interface and its options anymore. It's already too much stuff. The defaults should work fine for newcomers, but if you update anything you might have to set important new options.

I've updated the screenshot on the main site. You can orient yourself on those settings. Make sure you set the 50 in the _posts per page_ textbox. I've also updated the description of the release a bit to make it more concise.

And yap, strg is equal to ctrl. Sorry for the confusion. In Germany its called strg on the key (for Steuerung == control). I probably didn't pay attention to it

**Comment by tringate** [comment]
You were correct in that I had not put values into the new boxes in the existing blog details area.  They were actually blank.

However when using the CTRL-A hot key and setting them for all blogs, then adding them to the queue and doing a crawl only the first blog kept the settings and all the others went back to being blank.  I set each one individually, and then they persisted.  It should be noted that when I did the first crawl the program once again locked up the window and the only recourse was to use task manager to kill the job.  This only happens when the value for a blog is blank it appears.

Another thing I do not understand is that I checked the "force rescan" box expecting it to re-download all the files, but it did not do that.  It went through the blogs only downloading one or two files or none if none had been added to the blog since the last scan which was yesterday.

I also have download reblogs set in all my blogs if that makes any difference.

The reason I want to do the rescan is to get the actual post dates in all of the files rather than the date of when I did the scan which is more or less the old default for the majority of the files.

Do I have to delete the blog and scan it as if I have never downloaded it to make this happen?  I really want to get the file date to reflect when the original photo was posted as close as possible.  The only way I can accomplish that would be to be downloading the blog where it was first posted which is probably unlikely.

The ideal date of a file would be the original post date for a reblog file, but even using the date of the posted reblog will make sure I only have the very first post of a reblog should a file be reblogged in several of the blogs I download.

**Comment by johanneszab** [comment]
Rescan does not redownload. I don't think it makes sense to add a re-download option. It's probably easier to just delete and add that blog again.

With the .55 release you can export your blog urls in the settings under general. Then remove all the blog indices with the delete button or manually by delete the __Index__ folder in the download location. 

Re-add all blogs by selection everything in the exported .txt file. Copy it (ctrl-c) and the clipboard monitor will re-add all your blogs. If you still have the images, the re-download won't overwrite them, see them, skip them and set the date to the posts date. That's basically what you want I think. It just costs you the time to scan the blog again.

**Comment by johanneszab** [comment]
BTW: If you set something in the details for multiple blogs, you have to select any of the blogs in the manager to trigger the changes. Just select a random blog in your list, and all the saves will get applied. 

I see now that I could have implemented it differently.

**Comment by tringate** [comment]
I think the strange operation I was experiencing was because I left the .json files in the blog instance when I updated.  I finally deleted everything in the program directory removed the index file from the blog, moved all the photos to a "save" folder and then installed release 55.  It is downloading the entire blog as expected and is running very well.
In the past I never noticed those .json files in the folder so always deleted everything.  When I did leave those .json files it simply does not work right at all.

It is great having the file date be the post date.  This is going to help a bunch at removing duplicate files. 
 The blog with the original will be the one that has the file left.  If I only have files that were reblogged from some other blog I am not downloading, I will save the file in the first blog that did a reblog and all the later arrivals of that photo will end up getting deleted.

The only really desired feature I would like to see now would be to be able to save the blog folders into a "group" by adding an additional "group" folder name in the blog details screen for each blog.  That way all the cars blogs would be in the cars group sub folder and all the flags would be in the flags sub folder.  It's the same idea as in issue #50.

I can't think of a single other feature I would need to sort my photos into smaller groups and let me sort through smaller globs of photos to find those I was really interested in.

I have to say you have done a marvelous job with tumble three.  I am sorry I posted "non problems" caused by how I did the update this time.

I'm glad I asked the question about CTRL-A as I have been using it now many times.

The full screen photo preview was cool to watch too. They really come in fast, so it is like a movie sometimes.

Those pesky zero byte movies keep showing up too, and I believe it is because tumblr is deleting movies that violate their new TOS.  Before they would replace them with a notice, but now they delete the file and send the user a violation notice in an email.

I have to go through and delete all the zero bye files after a crawl.  I'm not sure if it downloads them again on a later crawl or not.  I have not looked into that, since it is easy to sort on file size and delete them all at once.
